recipe-server/client/control/components/RecipeFilters.js

```diff
@@ -128,7 +128,7 @@ export class RecipeFilters extends React.Component {
     return (
       <div className="fluid-8">
         <div id="secondary-header" className="fluid-8">
-          <div className="header-search" className="fluid-2">
+          <div className="header-search fluid-2">
```

Comment:
This changes the markup, because it used to render as just `class="fluid-2"`. I tested this, and fixed an issue that came up, relating to the display of the filter items when being hovered.